README.md

<!-- README.md is generated from README.Rmd. Please edit that file -->

# traumaR <img src='man/figures/logo.png' align="right" height="139" />

<!-- badges: start -->

[![](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://www.tidyverse.org/lifecycle/#experimental)
<!-- badges: end -->

The goal of traumaR is to provide users with human friendly results in a
normalized format.

## Installation

You can install the released version of traumaR from
[CRAN](https://CRAN.R-project.org) with:

``` r
install.packages("traumaR")
```

And the development version from [GitHub](https://github.com/) with:

``` r
# install.packages("devtools")
devtools::install_github("ultramattyice/traumaR")
```

## Example

This is a basic example which shows you how to solve a common problem:

``` r
library(traumaR)
library(kableExtra)

raw_data <- data.table::fread("E:/Northwestern/12 - Capstone/PTOS_Toy.csv", na.strings = c("<unk>", "", "<n/a>"))

traumaR::run_full_show(raw_data)
```

### Example of Patient Data Output

``` r
patient_df %>% 
  head(10) %>% 
  kable() %>% 
  kable_styling() %>% 
  scroll_box(width = "300px", height = "100px")
```

<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:100px; overflow-x: scroll; width:300px; ">

<table class="table" style="margin-left: auto; margin-right: auto;">

<thead>

<tr>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

id

</th>

<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">

trauma\_number

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

date\_of\_birth

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

age\_in\_yrs

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

sex

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

race

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

patient\_county

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

zip\_code

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

payor\_class\_primary

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

pay\_cat1\_o

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

primary\_injury\_type

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

injury\_dt

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

injury\_tm\_s

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

cause\_of\_injury\_e\_code

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

place\_of\_injury

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fall\_height

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

was\_the\_patient\_extricated

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

scene\_gcs\_eye

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

scene\_gcs\_verbal

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

scene\_gcs\_motor

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

transport\_gcs\_eye

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

transport\_gcs\_verbal

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

transport\_gcs\_motor

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

interhospital\_transport\_gcs\_eye

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

interhospital\_transport\_gcs\_verbal

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

interhospital\_transport\_gcs\_motor

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

referring\_facility\_gcs\_eye

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

referring\_facility\_gcs\_verbal

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

referring\_facility\_gcs\_motor

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

admission\_gcs\_eye

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

admission\_gcs\_verbal

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

admission\_gcs\_motor

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fasciotomy\_dt

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fasciotomy\_time

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fasciotomy\_loc

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fasciotomy\_cd

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fasciotomy\_desc

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

forearm\_fx\_dt

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

forearm\_fx\_time

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

forearm\_fx\_loc

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

forearm\_fx\_cd

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

forearm\_fx\_desc

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

code\_cd\_complication

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

code\_cd\_diagnosis

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

code\_cd\_procedure

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

code\_desc\_complication

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

code\_desc\_diagnosis

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

code\_desc\_procedure

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fltr\_diagnosis

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fltr\_procedure

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fltr\_complication

</th>

<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">

fltr\_fasciotomy

</th>

</tr>

</thead>

<tbody>

<tr>

<td style="text-align:left;">

0123198320000011

</td>

<td style="text-align:right;">

20000011

</td>

<td style="text-align:left;">

01/23/1983

</td>

<td style="text-align:left;">

16

</td>

<td style="text-align:left;">

MALE

</td>

<td style="text-align:left;">

Black

</td>

<td style="text-align:left;">

Delaware

</td>

<td style="text-align:left;">

19373

</td>

<td style="text-align:left;">

Commercial Insurer Indemnity

</td>

<td style="text-align:left;">

PRISON HEALTH C25

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

01/23/1983

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Fight/Brawl/Rape - Unarmed Fight or Brawl

</td>

<td style="text-align:left;">

Residential Institution

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

0208197920000001

</td>

<td style="text-align:right;">

20000001

</td>

<td style="text-align:left;">

02/08/1979

</td>

<td style="text-align:left;">

20

</td>

<td style="text-align:left;">

MALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Commercial Insurer Indemnity

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

02/08/1979

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Oth MVA Traffic, Highway Collision - Driver of MV, Non MC

</td>

<td style="text-align:left;">

Street/Highway

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

0527195820000003

</td>

<td style="text-align:right;">

20000003

</td>

<td style="text-align:left;">

05/27/1958

</td>

<td style="text-align:left;">

41

</td>

<td style="text-align:left;">

MALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

18914

</td>

<td style="text-align:left;">

Commercial Insurer Indemnity

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

05/27/1958

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Oth MVA Traffic, Highway Collision - Driver of MV, Non MC

</td>

<td style="text-align:left;">

Street/Highway

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

UNK

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

031219782000018

</td>

<td style="text-align:right;">

2000018

</td>

<td style="text-align:left;">

03/12/1978

</td>

<td style="text-align:left;">

21

</td>

<td style="text-align:left;">

FEMALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

Allegheny

</td>

<td style="text-align:left;">

15025

</td>

<td style="text-align:left;">

Commercial Insurer Indemnity

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

03/12/1978

</td>

<td style="text-align:left;">

03:00

</td>

<td style="text-align:left;">

MVA Traffic, Loss Control-No Collision - Driver of MV, Non MC

</td>

<td style="text-align:left;">

Street/Highway

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

None

</td>

<td style="text-align:left;">

Incomprehensible Sounds

</td>

<td style="text-align:left;">

Withdraws

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

1104196520000006

</td>

<td style="text-align:right;">

20000006

</td>

<td style="text-align:left;">

11/04/1965

</td>

<td style="text-align:left;">

34

</td>

<td style="text-align:left;">

MALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

Lawrence

</td>

<td style="text-align:left;">

16112

</td>

<td style="text-align:left;">

Self Pay

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

11/04/1965

</td>

<td style="text-align:left;">

00:01

</td>

<td style="text-align:left;">

MVA Traffic, Collision w/ Pedestrian - Pedestrian

</td>

<td style="text-align:left;">

Street/Highway

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

1016197220000017

</td>

<td style="text-align:right;">

20000017

</td>

<td style="text-align:left;">

10/16/1972

</td>

<td style="text-align:left;">

27

</td>

<td style="text-align:left;">

MALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

Allegheny

</td>

<td style="text-align:left;">

15601

</td>

<td style="text-align:left;">

Self Pay

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

10/16/1972

</td>

<td style="text-align:left;">

23:30

</td>

<td style="text-align:left;">

Suicide/Self Injury - Oth Spec Means

</td>

<td style="text-align:left;">

Public Building

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

None

</td>

<td style="text-align:left;">

None

</td>

<td style="text-align:left;">

None

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

0903192820000007

</td>

<td style="text-align:right;">

20000007

</td>

<td style="text-align:left;">

09/03/1928

</td>

<td style="text-align:left;">

71

</td>

<td style="text-align:left;">

FEMALE

</td>

<td style="text-align:left;">

Black

</td>

<td style="text-align:left;">

Philadelphia

</td>

<td style="text-align:left;">

19096

</td>

<td style="text-align:left;">

Medicare Indemnity

</td>

<td style="text-align:left;">

Medicare

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

09/03/1928

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

Fall on Same Level - Other

</td>

<td style="text-align:left;">

Home

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Confused

</td>

<td style="text-align:left;">

Localizes pain

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

01/02/2000

</td>

<td style="text-align:left;">

13:49

</td>

<td style="text-align:left;">

OR

</td>

<td style="text-align:left;">

79.32

</td>

<td style="text-align:left;">

Op red-int fix rad/ulna

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

0401197920000008

</td>

<td style="text-align:right;">

20000008

</td>

<td style="text-align:left;">

04/01/1979

</td>

<td style="text-align:left;">

20

</td>

<td style="text-align:left;">

MALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

Philadelphia

</td>

<td style="text-align:left;">

19036

</td>

<td style="text-align:left;">

Commercial Insurer Indemnity

</td>

<td style="text-align:left;">

Elect & Mgt Choice

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

04/01/1979

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

MVA Traffic, Collision w/ Pedestrian - Pedestrian

</td>

<td style="text-align:left;">

Street/Highway

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

01/02/2000

</td>

<td style="text-align:left;">

18:31

</td>

<td style="text-align:left;">

OR

</td>

<td style="text-align:left;">

79.32

</td>

<td style="text-align:left;">

Op red-int fix rad/ulna

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

<td style="text-align:left;">

FALSE

</td>

</tr>

<tr>

<td style="text-align:left;">

0816196020000001

</td>

<td style="text-align:right;">

20000001

</td>

<td style="text-align:left;">

08/16/1960

</td>

<td style="text-align:left;">

39

</td>

<td style="text-align:left;">

FEMALE

</td>

<td style="text-align:left;">

White

</td>

<td style="text-align:left;">

York

</td>

<td style="text-align:left;">

17313

</td>

<td style="text-align:left;">

Commercial Insurer Indemnity

</td>

<td style="text-align:left;">

———————-

</td>

<td style="text-align:left;">

Blunt

</td>

<td style="text-align:left;">

08/16/1960

</td>

<td style="text-align:left;">

20:02

</td>

<td style="text-align:left;">

Striking Against/Struck Accidentally - Oth w/ or w/o Subseq Fall

</td>

<td style="text-align:left;">

Home

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

NO

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Unknown

</td>

<td style="text-align:left;">

Spontaneous

</td>

<td style="text-align:left;">

Oriented

</td>

<td style="text-align:left;">

Obeys Command

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

NA

</td>

<td style="text-align:left;">

FALSE

</td>


NEWS.md
# traumaR 0.0.0.9500

* Added a `NEWS.md` file to track changes to the package.


R/COUNTY.R
#' The county of Pennsylvania in which the injury event took place
#'
#' @param col Indicate the county or state in which the injury took place using the 2 digit codes  provided.
#'
#' @return  State or County translation
#' @export
#'
county <- function(col) {
  col_value <-  case.(col == 1 , 'Adams' ,
                      col == 2 , 'Allegheny' ,
                      col == 3 , 'Armstrong' ,
                      col == 4 , 'Beaver' ,
                      col == 5 , 'Bedford' ,
                      col == 6 , 'Berks' ,
                      col == 7 , 'Blair' ,
                      col == 8 , 'Bradford' ,
                      col == 9 , 'Bucks' ,
                      col == 10 , 'Butler' ,
                      col == 11 , 'Cambria' ,
                      col == 12 , 'Cameron' ,
                      col == 13 , 'Carbon' ,
                      col == 14 , 'Centre' ,
                      col == 15 , 'Chester' ,
                      col == 16 , 'Clarion' ,
                      col == 17 , 'Clearfield' ,
                      col == 18 , 'Clinton' ,
                      col == 19 , 'Columbia' ,
                      col == 20 , 'Crawford' ,
                      col == 21 , 'Cumberland' ,
                      col == 22 , 'Dauphin' ,
                      col == 23 , 'Delaware' ,
                      col == 24 , 'Elk' ,
                      col == 25 , 'Erie' ,
                      col == 26 , 'Fayette' ,
                      col == 27 , 'Forest' ,
                      col == 28 , 'Franklin' ,
                      col == 29 , 'Fulton' ,
                      col == 30 , 'Greene' ,
                      col == 31 , 'Huntingdon' ,
                      col == 32 , 'Indiana' ,
                      col == 33 , 'Jefferson' ,
                      col == 34 , 'Juniata' ,
                      col == 35 , 'Lackawanna' ,
                      col == 36 , 'Lancaster' ,
                      col == 37 , 'Lawrence' ,
                      col == 38 , 'Lebanon' ,
                      col == 39 , 'Lehigh' ,
                      col == 40 , 'Luzerne' ,
                      col == 41 , 'Lycoming' ,
                      col == 42 , 'McKean' ,
                      col == 43 , 'Mercer' ,
                      col == 44 , 'Mifflin' ,
                      col == 45 , 'Monroe' ,
                      col == 46 , 'Montgomery' ,
                      col == 47 , 'Montour' ,
                      col == 48 , 'Northhampton' ,
                      col == 49 , 'Northumberland' ,
                      col == 50 , 'Perry' ,
                      col == 51 , 'Philadelphia' ,
                      col == 52 , 'Pike' ,
                      col == 53 , 'Potter' ,
                      col == 54 , 'Schuylkill' ,
                      col == 55 , 'Snyder' ,
                      col == 56 , 'Somerset' ,
                      col == 57 , 'Sullivan' ,
                      col == 58 , 'Susquehanna' ,
                      col == 59 , 'Tioga' ,
                      col == 60 , 'Union' ,
                      col == 61 , 'Venango' ,
                      col == 62 , 'Warren' ,
                      col == 63 , 'Washington' ,
                      col == 64 , 'Wayne' ,
                      col == 65 , 'Westmoreland' ,
                      col == 66 , 'Wyoming' ,
                      col == 67 , 'York' ,
                      col == 68 , 'Delaware' ,
                      col == 69 , 'Maryland' ,
                      col == 70 , 'New Jersey' ,
                      col == 71 , 'New York' ,
                      col == 72 , 'Ohio' ,
                      col == 73 , 'West Virginia' ,
                      col == 74 , 'Other State' ,
                      col == 75 , 'Virginia' ,
                      col == 79 , 'Other Country' ,
                      default = 'Unknown'
  )
  return(col_value)
}
